mongodbatlas/resource_mongodbatlas_cloud_backup_schedule.go

```diff
@@ -454,14 +454,16 @@ func resourceMongoDBAtlasCloudBackupScheduleImportState(ctx context.Context, d *
 }
 
 func cloudBackupScheduleCreateOrUpdate(ctx context.Context, conn *matlas.Client, d *schema.ResourceData, projectID, clusterName string) error {
+	policy := matlas.Policy{}
 	// Get policies items
 	resp, _, err := conn.CloudProviderSnapshotBackupPolicies.Get(ctx, projectID, clusterName)
 	if err != nil {
 		log.Printf("error getting MongoDB Cloud Backup Schedule (%s): %s", clusterName, err)
+	} else if len(resp.Policies) == 1 {
+		policy.ID = resp.Policies[0].ID
```

Comment:
Don't we want to do anything with id if we get more than 1 policy in the response?

Reply:
I just moved the existing logic that was on the bottom. I am not really sure why we only re-use the ID if there is only one policy.

@andreaangiolillo  do you have context here?

Reply:
policies can have only 1 element. Since this function is sued also for update, we need to reuse the policy id to update it as part of the Update operation

Reply:
@andreaangiolillo I figured that was gonna be the answer but funny enough I was not able to find where is the definition that says it only has 1 element. This is [the closest](https://github.com/mongodb/go-client-mongodb-atlas/blob/d023c4dda54a37f2d7b02242067e9398898e8843/mongodbatlas/cloud_provider_snapshot_backup_policies.go#L44) I found but I couldn't find where it says "only 1 elem".